internal/config/config.go

```diff
@@ -59,8 +61,13 @@ type Settings struct {
 }
 
 func defaultConfig(dataDir, configFile string) *Config {
+	forcePrepend := forcePrependDefault
+	forcePrependEnv := os.Getenv("ASDF_FORCE_PREPEND")
```

Comment:
We might want to be able to handle inconsistent case here. Some folks might set this to `YES` by mistake and I think that should be handled.

Reply:
I think it means case-insensitive. Sounds good to me.
I'll try to write a commit.